specification/security/resource-manager/readme.md

```diff
@@ -361,6 +372,20 @@ override-info:
   title: SecurityCenter
 ```
 
+### Tag: package-2020-08-preview-only
+
+These settings apply only when `--tag=package-2020-08-preview-only` is specified on the command line. This tag is used for Ruby SDK.
+
+``` yaml $(tag) == 'package-2020-08-preview-only'
+input-file:
+- Microsoft.Security/preview/2020-08-06-preview/iotDefenderSettings.json
+- Microsoft.Security/preview/2020-08-06-preview/iotSensors.json
```

Comment:
I assume you don't want to include the existing functionality in this version. So documentation and SDK will not show the existing functionality.

Reply:
Following the standard practice in `Microsoft.Security` to include only actual changes in new API versions.